src/app/Overview/Overview.tsx

```diff
@@ -159,6 +159,47 @@ const AggregateStatusCards: React.FunctionComponent = () => {
           layout: "multiIcon",
         }
       ],
+      accountsPerProvider: [
+        {
+          title: "AWS Accounts",
+          content: [
+            {
+              count: clusterCounts["AWS"] || 0,
```

Comment:
This code calculates the number of clusters per account.
In the `clusterPerCP.accounts.forEach((account)` loop we can have something like
```typescript
 accountsPerCP[provider] = (accountsPerCP[provider] || 0) + 1;
 ```
 and then use it
 ```typescript
 accountsPerProvider: [
      {
        title: 'AWS Accounts',
        content: [
          {
            count: accountsPerCP['AWS'] || 0,
            icon: <AwsIcon color="var(--pf-v5-global--danger-color--100)" />,
            ref: '/accounts?provider=AWS',
          },
        ],
        layout: 'multiIcon',
      }
 ```

Reply:
Fixed id this PR #15